packages/reason-relay/package.json

```diff
@@ -31,32 +31,32 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
     "bs-fetch": "^0.5.0",
-    "bs-platform": "^7.1.1",
+    "bs-platform": "^7.3.2",
     "graphql": "14.5.0",
     "graphql-query-test-mock": "^0.12.1",
     "jest": "^24.9.0",
     "nock": "^11.7.0",
     "node-fetch": "^2.6.0",
-    "react": "0.0.0-experimental-d28bd2994",
-    "react-dom": "0.0.0-experimental-d28bd2994",
-    "react-relay": "0.0.0-experimental-8cc94ddc",
-    "reason-react": "^0.7.0",
-    "relay-compiler": "9.0.0",
-    "relay-config": "9.0.0",
-    "relay-runtime": "9.0.0",
-    "reason-promise": "^1.0.2"
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
```

Comment:
These need to stay at the experimental versions they were (or possibly be updated, but if so, the experimental versions).

Reply:
Do you know why we need to have experimental version of `react`? I thought it was only because of concurrent mode? Or is it because  `relay@experimental` depends on `react@experimental`?